pom.xml

```diff
@@ -132,7 +132,7 @@
                     <version>${maven-release-plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <releaseProfiles>javadoc,deploy,release,docker-push</releaseProfiles>
+                        <releaseProfiles>javadoc,deploy,release,docker-push,-console</releaseProfiles>
```

Comment:
I've checked source code of the `maven-release-plugin` for the `release:perform` goal in [here](http://svn.apache.org/viewvc/maven/release/tags/maven-release-2.5.3/maven-release-plugin/src/main/java/org/apache/maven/plugins/release/PerformReleaseMojo.java?revision=1708589&view=markup) and it seems that the only reference to `releaseProfiles` is in a parameter that is referenced into `getAdditionalProfiles()`.

In the [AbstractReleaseMojo](http://svn.apache.org/viewvc/maven/release/tags/maven-release-2.5.3/maven-release-plugin/src/main/java/org/apache/maven/plugins/release/AbstractReleaseMojo.java) the method `getAdditionalProfiles()` is invoked in the `createReleaseDescriptor()`.

So I can say that this should work, as the string of `additionalProfiles` is now manipulated or stripped.

@ctron  do you agree on this?